### PR TITLE
Update versions for Release 9.2.0

### PR DESCRIPTION
**Pull request by chliangGoogle**
#no-changelog

### CODE REVIEW

File: FirebaseAnalyticsOnDeviceConversion.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
     s.name             = 'FirebaseAnalyticsOnDeviceConversion'
-    s.version          = '9.1.0'
+    s.version          = '9.2.0'
```
Comment (ncooke3) [inline FirebaseAnalyticsOnDeviceConversion.podspec:3]: Please update line 21 to instead be:
```ruby
s.dependency 'GoogleAppMeasurementOnDeviceConversion', '9.2'
```

**Review by ncooke3 — APPROVED** [review]

File: FirebaseAnalyticsOnDeviceConversion.podspec (hunk 1 above)
Comment (ncooke3) [reply FirebaseAnalyticsOnDeviceConversion.podspec:3]: Related to this discussion: https://github.com/firebase/firebase-ios-sdk/pull/9787#discussion_r868524192

Comment (chliangGoogle) [reply FirebaseAnalyticsOnDeviceConversion.podspec:3]: Done.

Comment (ncooke3) [reply FirebaseAnalyticsOnDeviceConversion.podspec:3]: Thank you, to fix the CI, could you now pod repo push the `FirebaseAnalyticsOnDeviceConversion.podspec` (you can copy and paste it from the command that pushed it earlier).

Comment (chliangGoogle) [reply FirebaseAnalyticsOnDeviceConversion.podspec:3]: Done. Using the cmd in the doc.

Comment (ncooke3) [reply FirebaseAnalyticsOnDeviceConversion.podspec:3]: ty! [Looks good!](https://github.com/firebase/SpecsStaging/commit/fb262a1aaa6c8a84381d08798883552204d3f298) Sorry for the extra steps.

**Comment by chliangGoogle** [comment]
The failure doesn't seem to relate to this change.

**Review by paulb777 — APPROVED** [review]
Yep the failure was a buildcache flake. I restarted it.

This is good to merge.

**Comment by paulb777** [comment]
Can this be merged?

**Pull request merged**